websites/S/Stadia/dist/metadata.json

```diff
@@ -1,15 +1,21 @@
 {
   "author": {
-    "name": "Alexx",
-    "id": "604182077088202784"
+    "name": "Dark_Ville",
+    "id": "638080361179512853"
   },
+  "contributors": [
+   {
+      "name": "Alexx",
+      "id": "604182077088202784"
+    }
+  ],
   "url": "stadia.google.com",
   "description": {
     "en": "Stadia is a cloud gaming service operated by Google. It is advertised to be capable of streaming video games up to 4K resolution at 60 frames per second with support for high-dynamic-range, to players via the company's numerous data centers across the globe, provided they are using a sufficiently high-speed internet connection. It is accessible through the Google Chrome web browser on desktop computers, Pixel smartphones, supported smartphones from Samsung, OnePlus, Razer and Asus, as well as Chrome OS tablets and Chromecast for TV support.",
     "nl": "Stadia is een door Google geëxploiteerde cloudgamingdienst. Er wordt geadverteerd dat het videogames kan streamen tot 4K-resolutie bij 60 frames per seconde met ondersteuning voor high-dynamic-range, naar spelers via de talrijke datacenters van het bedrijf over de hele wereld, op voorwaarde dat ze een internetverbinding met voldoende hoge snelheid gebruiken. Het is toegankelijk via de Google Chrome-webbrowser op desktopcomputers, Pixel-smartphones, ondersteunde smartphones van Samsung, OnePlus, Razer en Asus, evenals Chrome OS-tablets en Chromecast voor tv-ondersteuning."
   },
   "service": "Stadia",
-  "version": "1.0.3",
+  "version": "1.0.4",
   "logo": "https://i.imgur.com/DwoRFyw.png",
   "thumbnail": "https://i.imgur.com/CyujQo7.png",
   "color": "#0081FF",
```

Comment:
Well about stadia - someone else is going to do it - so basically revert changes for now.